storage/pipeline/states_failed.go

```diff
@@ -301,8 +301,21 @@ func (m *Sealing) handleCommitFailed(ctx statemachine.Context, sector SectorInfo
 
 		switch mw.Receipt.ExitCode {
 		case exitcode.Ok:
-			// API error in CcommitWait
-			return ctx.Send(SectorRetryCommitWait{})
```

Comment:
Oh woah we were just permanently looping if cron activation failed?

Reply:
Correct.. Fortunately "just" waiting for the message, not re-submitting